oauth2client/service_account.py

```diff
@@ -683,3 +666,44 @@ def _create_token(self, additional_claims=None):
         jwt = crypt.make_signed_jwt(self._signer, payload,
                                     key_id=self._private_key_id)
         return jwt.decode('ascii'), expiry
+
+    def apply(self, headers, access_token=None):
+        """Add the access token as the Authorization header in headers.
+
+        Args:
+            headers: dict, the headers to add the Authorization header to.
+            access_token: An access token to use instead of the credential's
+                current access token.
+        """
+        if not access_token:
+            access_token = self.access_token
+        headers['Authorization'] = 'Bearer ' + access_token
```

Comment:
I happen to work on [an HTTP/2-based library](http://www.grpc.io/) and recently learned in the course of a painful user-visible outage that [in HTTP/2 all header names must contain no majuscule characters](http://http2.github.io/http2-spec/index.html#rfc.section.8.1.2). Can this header name be "authorization"? Do we support use of `oauth2client` with HTTP/2? How aware are we that `oauth2client` is already in use with HTTP/2 whether we support it or not?